app/components/createContainer/createContainerController.js

```diff
@@ -179,6 +179,7 @@ function ($q, $scope, $state, $stateParams, $filter, Container, ContainerHelper,
     var networkMode = mode;
     if (containerName) {
       networkMode += ':' + containerName;
+      config.Hostname = '';
```

Comment:
When you use ither container network as your network, you use its hostname too.
So docker Api does'nt support both options at same time